network-api/networkapi/wagtailpages/templates/maintenance/maintenance.html

```diff
@@ -27,11 +27,6 @@
         <div class="takeover"></div>
         <div class="wrapper">
             <div class="sticky-top">
-                <div class="wrapper-join-us">
```

Comment:
Same. We never load our compiled JS file into this page which means the old React component for rendering newsletter sign form never worked.